### DISCUSSION TITLE
请问如何AMLL桌面版构建到本地？

### DISCUSSION THREAD

**Discussion by Hamilton-Liu** [amll-dev/applemusic-like-lyrics#293] [Q&A] [answered]
请问如何AMLL桌面版构建到本地？ yaml格式的配置文件不知道怎么操作，请大佬赐教

**Comment by FinaFina233** [comment]
Actions → Workflows → 构建 AMLL Player Tauri 桌面版本 → 点击最新的run → 翻到底部 → Artifacts

**Comment by Linho1219** [comment, marked as answer]
您可以直接从如上所述的 actions 下载构建结果，也可以从主仓库的 [release](https://github.com/amll-dev/applemusic-like-lyrics/releases/tag/main-player-dev) 处下载。

AMLL Player 由于包含 ffmpeg 等依赖，构建较为繁琐。简言之，您需要在本地配置：

- node
- pnpm
- rustc/cargo
- wasm32-unknown-unknown target
- wasm-pack
- tauri-cli
- FFmpeg 预编译库的发现配置
- pkg-config/vcpkg

配置后运行：

```sh
pnpm nx build player
pnpm -F player tauri build
```

希望以上内容有帮助。
